From c769bd359689c0d3ee9484d26c90a390030bd614 Mon Sep 17 00:00:00 2001
From: Tram Nguyen <tram.nguyen.xw@rvc.renesas.com>
Date: Thu, 8 Sep 2016 12:01:23 +0700
Subject: [PATCH 1/3] gst145-rtpL16depay: Allow arbitrary payload-id

Porting from a patch in Linux 3.10 environment made by Petr Nechaev <petr.nechaev@cogentembedded.com>

Signed-off-by: Tram Nguyen <tram.nguyen.xw@rvc.renesas.com>
---
 gst/rtp/gstrtpL16depay.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gst/rtp/gstrtpL16depay.c b/gst/rtp/gstrtpL16depay.c
index 5bbbedc..a745627 100644
--- a/gst/rtp/gstrtpL16depay.c
+++ b/gst/rtp/gstrtpL16depay.c
@@ -70,8 +70,8 @@ static GstStaticPadTemplate gst_rtp_L16_depay_sink_template =
         "encoding-name = (string) \"L16\";"
         "application/x-rtp, "
         "media = (string) \"audio\", "
-        "payload = (int) { " GST_RTP_PAYLOAD_L16_STEREO_STRING ", "
-        GST_RTP_PAYLOAD_L16_MONO_STRING " }," "clock-rate = (int) [ 1, MAX ]"
+        "payload = (int) [1, 255],"
+        "clock-rate = (int) [ 1, MAX ]"
         /* "channels = (int) [1, MAX]" */
         /* "emphasis = (string) ANY" */
         /* "channel-order = (string) ANY" */
--
1.9.1
